README.md

```diff
@@ -1,5 +1,208 @@
+
+<div style="text-align:center">
+<img align="center" src="./dosc/media/../../docs/media/api-governance-processes-steps.png" width="600">
+</div>
+<p>&nbsp;</p>
+
+### Step 1: Create API Specification
+Use your preferred editor to create a YAML file for the API specification and store it in your versioning repository (ex. git). Ensure each specification has an API identifier (`x-api-identifier`) with a unique name. _The API specifications will be loosely coupled, no server part and only one specific information object (resource)._ 
+
+  ```sh
+    openapi: 3.0.0
+    info:
+      description: This is a simple API for accessing temperature sensor data.
+      title: Temperature Sensor API HTTP(S)-JSON
+      version: 1.0.0
+      x-service-identifier: temperature-sensor
+    paths:
+      /temperature:
+        get:
+          responses:
+            200:
+              content:
+                application/json:
+                  schema:
+                    $ref: '#/components/schemas/TemperatureData'
+              description: Temperature data
+          summary: Get current temperature data
+          x-accepts: application/json
+  ```
+
+### Step 2: Publish API Specification
+Utilize the approval CLI Tool to publish your API specification to the API specification repository. _Alternative: Use example pipelines matching your environment_.
+For test and deemonstration purpose use the command line call publish the specification.
+
+  ```sh
+  npx @sinetiq/api-specification-registry-cli --glob **/api-specification.json --ignore-already-registered --file api-spec.yml
+  ```
+
+### Verification
+Check the API specification registry UI to confirm the correct approval and deployment of your API specifications.
+
+1. Start a web browser
+2. Type in `api-registry-url:3333`
+3. Your spec is now published within the API specification registry.
+
+### Step 3: Application Development
+Develop applications, both providers and consumers, that comply with the API specification. Provider should publish endpoints with the unique API identifier, and consumers should discover and connect to providers using the identifier.
+
+1. Service Provider(s) publish the endpoint, for example serviceType `temperature-service-rest-json`.
+
+    ```sh
+    ServiceData sd = new ServiceData();
+    sd.setHost(host);
+    sd.setPort(port);
+    String serviceName = environment.getProperty("service.name");
+    String serviceType = environment.getProperty("service.type");
+    sd.setName(new ServiceName(serviceName, new ServiceType(serviceType)));
+    String basePath = environment.getProperty("service.basePath");
+    sd.getProperties().put("path", basePath);
+    consulAPI.registerService(sd);
+    ```
+    _See complete Provider example code here._
+
+2. Service Consumer(s) discover the information source endpoint to be able to establish connection.
+
+    ```sh
+    ServiceType serviceType = new ServiceType("temperature-sensor-rest-json");
```

Comment:
Same here, use correct coordinate.

Also, a thought we can take later. Here we talk about service types, which we kind of abandoned as a term.